sessions.md

```diff
@@ -0,0 +1,210 @@
+
+- The Expiration time each cookie should be determined by the `exp` claim of
+the token, and this value is controlled by the Oauth Policy of the Application
+
+**NOTE**: We do NOT need to support server-side session storage, because the way
+this works is like so: a JWT is created which contains an account href.  The
+account href is persisted in our internal Stormpath SDK cache.  This means no
+server side session storage is necessary as the caching optimizations are
+already handled internally (*by us*).
+
+
+## <a name="Options"></a> Options
+
+This table is a list of all the options that are required by this feature.
+Detailed descriptions follow.  How the option names are translated into the
+framework language (e.g. to camel case, or not)? Is not specified here.
+
+```json
+{
+  "session": {
+    "httpOnly": true,
+    "secure": null,
+    "path": "/",
+    "domain": null,
+    "validationStrategy": "stormpath"
+  }
+}
+```
+
+#### <a name="http-only"></a> httpOnly
+
+This option determines whether or not this cookie can be read by Javascript code
+on the browser.
+
+**NOTE**: Disabling this option is a potential security issue, as your access
+tokens could be hijacked via XSS.
+
+<a href="#top">Back to Top</a>
+
+
+#### <a name="secure"></a> secure
+
+This option determines whether or not our cookies can be set over a non-https
+connection.  By default, this option is null, which means we will attempt to
+inspect the protocol of the incoming request, and make a decision of what to do
+automatically.
+
+Defining this value forces the `secure` mode to be either enabled or disabled
+all the time.
+
+**NOTE**: Disabling this option is a potential security issue, as setting
+cookies over a non-https connection is ripe for traffic inspection attacks.
+
+<a href="#top">Back to Top</a>
+
+
+#### <a name="path"></a> path
+
+This option determines what URI paths will be able to read our cookies.  By
+default, this is set to the root path (`/`), meaning our cookies will be
+available to every URI on the web server.
+
+<a href="#top">Back to Top</a>
+
+
+#### <a name="domain"></a> domain
+
+This option determines what domain(s), can read this cookie value.  By default,
+this is `null`, meaning only the current domain where the cookie is set can use
+read this cookie value.
+
+The only time you might want to set this value is if you want to allow
+subdomains to access your cookies as well.
+
+<a href="#top">Back to Top</a>
+
+
+#### <a name="validation-strategy"></a> validationStrategy
+
+Determines how we validate an access token.  There are three strategies:
+
+* **local** - only verify the signature, expiration, and issuer
+* **stormpath** - always validate against Stormpath, for additional checks
+such as Account and Application status.  This is the default, but the most
+consistent (will always be aware of Stormpath resource statuses).
+* **random** - 50/50 chance, per request, of **local** or **stormpath** being used.
```

Comment:
`random` has been dropped in #12 

`local` and `stormpath` are directives that tell the SDK to use (or not use) the `withLocalValidation()` option on the authentication builder.

In #12 this is now defined as an array of strings.  If the user wants to supply a custom validator, they could add it to the list (or have it be the only one in the list).

If you want to continue discussing, please start a new thread on #12